schema-supervisory-authorities.json

```diff
@@ -0,0 +1,110 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Datenanfragen.de supervisory authority database record",
+	"description": "A single record of the Datenanfragen.de supervisory authority database. It represents the contact information of an authority specifically for lodging complaints.\nMore information and access to the whole database at: https://github.com/datenanfragen/companies",
+	"type": "object",
+	"properties": {
+		"slug": {
+			"type": "string",
+			"title": "Slug",
+			"pattern": "^[a-z0-9-]+$",
+			"description": "A unique identifier for the company record. Also defines the filename."
+		},
+		"relevant-countries": {
+			"type": "array",
+			"title": "Relevant countries",
+			"description": "A list of countries the record is relevant for. The countries can either be an ISO 3166-1 alpha-2 country code or 'all'. If a company has branches in different countries with unique contact data, separate records should be created for those. If a company branch offers its services in English and isn't explicitly restricted to certain countries, 'all' should be specified.",
```

Comment:
The description is not appropriate here.

Also, I am not sure if this needs to be an array. Are there any SVAs relevant to more than one country? There might be but I can't think of any right now.

Reply:
Resolved.